app/templates/tools/includes/list.html

```diff
@@ -37,7 +37,7 @@ <h2 class="heading-medium">Your tools</h2>
         </td>
       </tr>
       {% endfor %}
-    {% else %}
+
```

Comment:
If you remove the `if`/`else`, doesn't always loop through the tools and then shows the RStudio row (potentially twice)?

Maybe I'm missing something.